Free-Code-Camp-JavaScript-Style-Guide.md

```diff
@@ -30,7 +31,7 @@ function foo() {
 }
 
 ```
-Bad: 
+Bad:
 ```js
 var foo = function () {

```

Comment:
I think this is a wrong bad example.  The curly brace should be on the next line.